tests/framework/test_values.py

```diff
@@ -55,15 +62,15 @@ def test_replace_combiner(manager):
     assert value() == 84
 
 
-def test_joint_value(manager):
+def test_joint_value(manager: ValuesManager) -> None:
     # This is the normal configuration for PAF and disability weight type values
     index = pd.Index(range(10))
 
     value = manager.register_value_producer(
         "test",
         source=lambda idx: [pd.Series(0.0, index=idx)],
         preferred_combiner=list_combiner,
-        preferred_post_processor=union_post_processor,
+        preferred_post_processor=union_post_processor,  # type: ignore [arg-type]
```

Comment:
The PostProcessor class takes Any as an arg and outputs Any and union_post_processor is more specific using our generic Numberlike type which makes mypy very angry.